warehouse/models/mart/gtfs_schedule_latest/dim_stops_latest.sql

```diff
@@ -4,6 +4,52 @@ dim_stops_latest AS (
     table_name = ref('dim_stops'),
     clean_table_name = 'dim_stops'
     ) }}
+),
+
+stg_state_geoportal__state_highway_network_stops AS (
+SELECT *
+-- FROM `cal-itp-data-infra-staging.external_state_geoportal.stg_state_geoportal__state_highway_network_stops`
+FROM {{ ref('stg_state_geoportal__state_highway_network_stops') }}
+),
+
+
+buffer_geometry_table AS (
+    SELECT
+        ST_BUFFER(wkt_coordinates,
+            30.48) AS buffer_geometry
```

Comment:
**suggestion:** This is 100ft, right? Could we add a short comment that says so? Even better if it provides context for why we're using 100ft (even if it's just "somebody decided that within 100 ft of the highway meant on the highway network.")

Reply:
yes great point – added a comment